src/index.js

```diff
@@ -1,13 +1,13 @@
 var isArray = Array.isArray
 
 export default function cc(obj) {
-  var out = ""
```

Comment:
`var`s are hoisted, so it doesn't matter where we declare it. Please put this one back where it was.

Reply:
Hoisting is for declaration, not assignment, that's why I've put it there.

Reply:
Let's use `let` to initialize it then.

Reply:
Using `let` here will drop IE < 11 support, let me know if that's ok.

Reply:
IE 11? What's that? Just kidding. Worth it IMO. 🙆‍♂

Reply:
Definitely ok.👌